community.md

```diff
@@ -18,7 +18,7 @@ Some communities have strong preferences for particular licenses. If you want to
 * [GNU](https://www.gnu.org/licenses/license-recommendations.html) recommends [GNU GPLv3](/licenses/gpl-3.0/) for most programs
 * [npm packages](https://libraries.io/search?platforms=npm) overwhelmingly use the [MIT](/licenses/mit/) or the very similar [ISC](/licenses/isc) licenses
 * [OpenBSD](https://www.openbsd.org/policy.html) prefers the [ISC License](/licenses/isc/)
-* [Rust](https://rust-lang.github.io/api-guidelines/necessities.html#crate-and-its-dependencies-have-a-permissive-license-c-permissive) crates are overwhelmingly licensed under both [MIT](/licenses/mit/) and [Apache License 2.0](/licenses/apache-2.0/)
+* [Rust](https://rust-lang.github.io/api-guidelines/necessities.html#crate-and-its-dependencies-have-a-permissive-license-c-permissive) crates are overwhelmingly dual-licensed under both [MIT](/licenses/mit/) and [Apache License 2.0](/licenses/apache-2.0/) at the crate user's option
```

Comment:
I don't think "dual-" or "at the crate user's option" are really comprehensible to the beginner, which is the main target audience. The link explains it pretty well, but here's an idea for increasing accuracy without using confusing works, but further inviting reader to learn more:

```suggestion
* [Rust](https://rust-lang.github.io/api-guidelines/necessities.html#crate-and-its-dependencies-have-a-permissive-license-c-permissive) crates are overwhelmingly licensed [MIT](/licenses/mit/) `[OR](https://spdx.github.io/spdx-spec/v2.3/SPDX-license-expressions/#d42-disjunctive-or-operator)` [Apache License 2.0](/licenses/apache-2.0/)
```

Reply:
Makes sense, thanks!